Mininet Emulator Framework & Ryu SDN Controller Setup

Vamsi995 opened the pull request:
- Mininet Emulator Framework
  - Custom Full Mesh Topology Support
  - Dynamic Link State Setup
 
- Ryu SDN Controller
  - Dynamic Path Switching logic
  - Latency Measurement Processor

pradeeban commented:
Good work, @Vamsi995.

Only one Git best practice. Can you please delete the folder "__pycache__" folder from your commits in dev branch? After you delete it, add __pycache__ to your .gitignore so that you won't mistakenly add and commit it again (and then update this pull request?)

We need to keep the Git to be minimal, free from binary and temp files.

Vamsi995 commented:
> Good work, @Vamsi995.
> 
> Only one Git best practice. Can you please delete the folder "**pycache**" folder from your commits in dev branch? After you delete it, add **pycache** to your .gitignore so that you won't mistakenly add and commit it again (and then update this pull request?)
> 
> We need to keep the Git to be minimal, free from binary and temp files.

I removed it @pradeeban. I added it to the gitignore file.

pradeeban commented:
Thanks, merged!
